fitparse/processors.py

```diff
@@ -32,36 +39,38 @@ def _scrub_method_name(self, method_name):
             method_name: Method name to scrub.
 
         Returns:
-            Scrubbed method name.
+            Scrubbed method (bounded).
         """
-        if method_name not in self._scrubbed_method_names:
-            self._scrubbed_method_names[method_name] = (
-                scrub_method_name(method_name))
+        method = self._method_cache.get(method_name, False)
+        if method is not False:
+            return method
+
+        scrubbed_method_name = scrub_method_name(method_name)
+        try:
```

Comment:
Can do `method = getattr(self, scrubbed_method_name, None)` to use `None` as the default if the attribute doesn't exist.

Reply:
`getattr` is slow for large FITs (e.g. 10 hours records). It's the purpose of this commit to avoid it and speed the processors.

Reply:
I understand that. I'm suggesting using `getattr(self, scrubbed_method_name, None)` instead of the `try`/`except` for the initial caching.